projects/dart/features/environment.py

```diff
@@ -8,10 +8,17 @@
 from webdriver_manager.phantomjs import PhantomJsDriverManager
 
 
-BEHAVE_DEBUG_ON_ERROR = True
+BEHAVE_DEBUG_ON_ERROR = os.getenv('BEHAVE_PDB', '')
 WAIT = os.getenv('WAIT', 1)
 
 
+def before_all(context):
+    context.credentials = {
+        'login': 'moonmoon',
+        'pass': 'm00n123!Moon'
```

Comment:
Do we need a `pass`? I think our current backend doesn't support it. Is this in anticipation of HarvardKey support in the future?

Reply:
Yeah, I had originally created the `pass` key when I was pulling in HarvardKey credentials from an environment variable. It's continued inclusion was a just-in-case sort of thing RE: HarvardKey. In light of our discussion of 2FA with HarvardKey, I'll go ahead and take this stuff out.